Split email guides, use Micronaut Email (#750)
micronaut-projects/micronaut-guides: guides/micronaut-email/controller.adoc

=== Controller

> Create `MailController` class. It uses a collaborator, `emailSender` to send and email.

Reviewer: ```suggestion
Create a `MailController` class. This class uses a collaborator, `emailSender`, to send an email.
```

> You can send emails asynchronously using https://micronaut-projects.github.io/micronaut-email/latest/api/index.html[AysnEmailSender] API or synchronously using the https://micronaut-projects.github.io/micronaut-email/latest/api/index.html[EmailSender] API.

Reviewer: ```suggestion
You can send emails asynchronously using the https://micronaut-projects.github.io/micronaut-email/latest/api/index.html[AysnEmailSender] API or synchronously using the https://micronaut-projects.github.io/micronaut-email/latest/api/index.html[EmailSender] API.
```
